src/components/pages/dashboard/dashboard.js

```diff
@@ -385,6 +387,7 @@ export class Dashboard extends Component {
               telemetry={telemetry}
               isPending={telemetryIsPending}
               error={deviceGroupError || telemetryError}
+              theme={theme}
               colors={chartColorObjects}
```

Comment:
Shouldn't the colors come from the theme?

Reply:
They should but that is a larger item than this task. Currently, if you switch to the light theme, the tsi charts stay on the default dark theme where the legends and axis are white. So you can't see them, This PR just makes the charts readable when the light theme is turned on.